R/v.R

```diff
@@ -129,7 +129,7 @@ v <- function(
 
   # Round numeric columns to 3 decimal places
   if(!is.null(.digits)){
-    .df <- .df %>% dplyr::mutate(across(where(is.numeric),\(x) prettyNum2(x, .digits)))
+    .df <- .df %>% dplyr::mutate(across(where(is.numeric), ~prettyNum2(.x, .digits)))
```

Comment:
I initially had it this way, but changed it to an anonymized function after getting some warning messages during `devtools::test()`. If you run `pkgr install`, do you get these warnings?

Reply:
If not happy to approve